src/BloomExe/WebLibraryIntegration/BookTransfer.cs

```diff
@@ -389,6 +390,25 @@ public string UploadBook(string bookFolder, IProgress progress, out string parse
 						Analytics.Track("UploadBook-Failure", new Dictionary<string, string>() { { "url", metadata.BookOrder }, { "title", metadata.Title }, { "error", e.Message } });
 					return "";
 				}
+				catch (AmazonS3Exception e)
+				{
+					if (e.Message.Contains("The difference between the request time and the current time is too large"))
+					{
+						progress.WriteError(LocalizationManager.GetString("PublishTab.Upload.TimeProblem",
+							"There was a problem uploading your book. This is probably because the system time on your computer is not correct for the timezone the computer is set to."));
+						if (!UseSandbox)
```

Comment:
> "system time on your computer is not correct for the timezone the computer is set to" 

I can't figure out what that means, and wouldn't know how to fix it. Is there a link we can point them too, or something to google?